app/Http/Controllers/TransactionsController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Jobs\TransactionImporter;
+use App\Models\Transaction;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Session;
+
+class TransactionsController extends Controller
+{
+    protected $user;
+
+    public function __construct()
+    {
+        // Could add middleware here, but adding instead on web.php for code cleanliness.
+
+        // Middleware : Auth
+
+        $this->middleware(function ($request, $next) {
+            $this->user = Auth::user();
+
+            return $next($request);
+        });
+    }
+
+    public function index()
+    {
+        $transactions = Transaction::where('user_id', $this->user->id)->orderBy('date', 'DESC')->get();
```

Comment:
```
public function index(Request $request)
    {
        $transactions = $request->user()->transactions()->orderBy('date', 'DESC')->get();
```

Reply:
Totally oversight on my part... fixed in `045d3b6`